.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
No please

Reply:
I think this might defeat the usefullness of the template ? 🤔 
The goal is to ultimately help community members to pause and try to produce a detailed enough bug report that is actionable by the project maintainers

Otherwise, people can abuse this

Some other examples of popular repos which have this value to "false"
- SolidJS (15k github stars): https://github.com/solidjs/solid/blob/main/.github/ISSUE_TEMPLATE/
- G6 - (9k Github stars): https://github.com/antvis/G6/tree/master/.github/ISSUE_TEMPLATE
- Tinybase (1k stars): https://github.com/tinyplex/tinybase/tree/main/.github

This is the documentation for the `blank_issues_enabled`
- https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

Reply:
Yes, the goal is to get people to pause and consider their issue, and to give them a template to fill out in case they don't know what information they should include.

> Otherwise, people can abuse this

It's not like this template won't help people who are simply confused regardless. People who go out of their way to abuse the system can be blocked. It's not like they'd be any less of a nuisance even if blank issues were disabled.

In my eyes, disabling blank issues only hurts, not helps. Remember, the users aren't the enemy, they just need help.

> Some other examples of popular repos which have this value to "false"

This isn't a good argument.

Reply:
Ok 👌  I will change to `false`

Reply:
Updated 👌 
- commit: https://github.com/caplin/FlexLayout/commit/5d3bbe9db6e3430e671512ad57d8ddaa543503c9
- demo: https://github.com/cliffordfajardo/FlexLayout/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

Reply:
FWIW, SolidJS (your first example) only just added templates, and there was at least one complaint on Discord about the inability to open blank issues. Templates can handle most cases but can't imagine every possible type of issue. Thanks to this PR and comment I learned about the ability to set `blank_issues_enabled: true` and made a PR to SolidJS to do so. So thanks for teaching me about it! And I agree that users should be allowed to choose what they want to do.

Reply:
@edemaine -- ahh thanks for the context and story about `blank_issues_enabled: true`
- https://github.com/solidjs/solid/pull/868#issue-1149713822